_posts/2018-11-24-why-would-a-java-engineer-love-fronted-development.md

```diff
@@ -190,15 +190,15 @@ from CLI.
 **Fast updates with minimal breaking changes**
 
 In the Java community, we are used to the fact that a code written 10 years ago
-will work on a new JVM. There are some exceptions like transition from Java 8 
-to Java 9, but generally, it's true. On the other hand, in frontend tendencies 
-are different: you take a break for 6 months and boom, everything is changed,
+will work on a new JVM. There are some exceptions like when transitioning from Java 8 
```

Comment:
I would rephrase: "*...when transitioning from Java 8 to Java 9*".

Will it sound correct if I change it to one of the following options:
- "There are some exceptions like migration from Java 8 to Java 9, but generally, it's true"
- "There are some exceptions like update from Java 8 to Java 9, but generally, it's true"
- "It is not always true - there were many problems migrating projects from Java 8 to Java 9"

I hope this will give you some context.

Reply:
The first two are almost correct:
"There are some exceptions, like with migration from Java 8 to Java 9, but generally, it's true."
"There are some exceptions, like when updating from Java 8 to Java 9, but generally, it's true."
The last one is not correct. You could use either of the first 2 options.